<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextDrawingToolMenu</class>
 <widget class="QWidget" name="TextDrawingToolMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1187</width>
    <height>53</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Color: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ColorPickerLabel" name="colorLabel">
     <property name="styleSheet">
      <string notr="true">ColorPickerLabel{
	margin: 3.5px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Font: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFontComboBox" name="fontComboBox"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Font size: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="sizeSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>30</number>
     </property>
     <property name="displayIntegerBase">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="boldCheckBox">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Bold</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="italicCheckBox">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Italic</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="underlineCheckBox">
     <property name="font">
      <font>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Underline</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cancelButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: red;
border:none;
color: white;
padding: 5px;</string>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="finishButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: green;
border:none;
color: white;
padding: 5px;</string>
     </property>
     <property name="text">
      <string>Finish writing</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorPickerLabel</class>
   <extends>QLabel</extends>
   <header location="global">colorpickerlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
